hotel-bookings-api/server.js

```diff
@@ -1,20 +1,63 @@
+process.env.PORT = process.env.PORT || 3001;
 const express = require("express");
 const cors = require("cors");
-
 const app = express();
-
 app.use(express.json());
 app.use(cors());
 
```

Comment:
Could you please tell me why `cors` has been installed here?

Reply:
The CORS library is needed to let my React frontend talk to your Node.js backend without security issues.